### ISSUE TITLE
cannot find symbol Image_picker

### ISSUE THREAD

**Issue by almazamangaliev** [flutter/flutter#33660]
## Logs

/Users/admin/Documents/FlutterApp/zarulem_app/android/app/src/main/java/io/flutter/plugins/GeneratedPluginRegistrant.java:17: error: cannot find symbol
import io.flutter.plugins.imagepicker.ImagePickerPlugin;
                                     ^
  symbol:   class ImagePickerPlugin
  location: package io.flutter.plugins.imagepicker
/Users/admin/Documents/FlutterApp/zarulem_app/android/app/src/main/java/io/flutter/plugins/GeneratedPluginRegistrant.java:18: error: cannot find symbol
import com.lyokone.location.LocationPlugin;
                           ^
  symbol:   class LocationPlugin
  location: package com.lyokone.location
/Users/admin/Documents/FlutterApp/zarulem_app/android/app/src/main/java/io/flutter/plugins/GeneratedPluginRegistrant.java:47: error: cannot find symbol
    ImagePickerPlugin.registerWith(registry.registrarFor("io.flutter.plugins.imagepicker.ImagePickerPlugin"));
    ^
  symbol:   variable ImagePickerPlugin
  location: class GeneratedPluginRegistrant
/Users/admin/Documents/FlutterApp/zarulem_app/android/app/src/main/java/io/flutter/plugins/GeneratedPluginRegistrant.java:48: error: cannot find symbol
    LocationPlugin.registerWith(registry.registrarFor("com.lyokone.location.LocationPlugin"));
    ^
  symbol:   variable LocationPlugin
  location: class GeneratedPluginRegistrant
4 errors

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:compileDebugJavaWithJavac'.

[✓] Flutter (Channel master, v1.6.4-pre.14, on Mac OS X 10.14.1 18B75, locale ru-KZ)
    • Flutter version 1.6.4-pre.14 at /Users/admin/Documents/FlutterApp/flutter
    • Framework revision 81c38b22cb (7 days ago), 2019-05-24 22:51:02 -0400
    • Engine revision 8dc3a4cde2
    • Dart version 2.3.2 (build 2.3.2-dev.0.0 e3edfd36b2)

 
[✓] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
    • Android SDK at /Users/admin/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-28, build-tools 28.0.3
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1136-b06)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 10.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 10.1, Build version 10B61
    • CocoaPods version 1.5.3

[✓] iOS tools - develop for iOS devices
    • ios-deploy 1.9.4

[✓] Android Studio (version 3.2)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 31.1.1
    • Dart plugin version 181.5656
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1136-b06)

[✓] Connected device (2 available)
    • Android SDK built for x86 • emulator-5554                            • android-x86 • Android 8.1.0 (API 27) (emulator)
    • Almaz                     • 042cc67b0b4f1661533c9c912c31df1ece93d66e • ios         • iOS 12.1.4

**Comment by BondarenkoStas** [comment]
@almazamangaliev did you find a solution? is it the same with current flutter stable and current image_picker?

**Comment by almazamangaliev** [comment]
yes, you need migrate project to androidX or use plugin dependencies in your pubspec.yaml to the last major version https://flutter.dev/docs/development/packages-and-plugins/androidx-compatibility

**Comment by almazamangaliev** [comment]
> @almazamangaliev вы нашли решение? то же самое с текущим стабильным флаттера и текущим image_picker?

да, вам нужно перенести проект в androidX или использовать зависимости плагинов в вашем pubspec.yaml до последней основной версии https://flutter.dev/docs/development/packages-and-plugins/androidx-compatibility

**Comment by BondarenkoStas** [comment]
Good, than I think it's safe to close the issue.
